conformance/lang/expressions/string-template-expr/string_template_expr.balt

```diff
@@ -0,0 +1,323 @@
+Test-Case: output
+Description: Test whether string-template-expr interpolates the results of evaluating expressions into a literal string.
+Labels: string, function-defn, int, string-template-expr, string-literal
```

Comment:
Do we need `function-defn` label in here. We do not define a new function inside the test-case.